Tools/WebKitTestRunner/cocoa/TestControllerCocoa.mm

```diff
@@ -199,15 +198,15 @@ VKImageAnalysisRequestID swizzledProcessImageAnalysisRequest(id, SEL, VKCImageAn
 
     [WKContentRuleListStore _setContentRuleListStoreForResourceMonitorURLsControllerForTesting:[WKContentRuleListStore storeWithURL:url.get()]];
 
-#if PLATFORM(MAC)
-    // See NetworkProcess::platformInitializeNetworkProcessCocoa for supporting a local DNS resolver on iOS.
+#if ENABLE(DNS_SERVER_FOR_TESTING) && !ENABLE(DNS_SERVER_FOR_TESTING_IN_NETWORKING_PROCESS)
+    // See NetworkProcess::platformInitializeNetworkProcessCocoa for supporting a local DNS resolver when ENABLE(DNS_SERVER_FOR_TESTING_IN_NETWORKING_PROCESS) == 1.
```

Comment:
"when ENABLE(DNS_SERVER_FOR_TESTING_IN_NETWORKING_PROCESS) == 1" => "when ENABLE(DNS_SERVER_FOR_TESTING_IN_NETWORKING_PROCESS)" or  "when ENABLE(DNS_SERVER_FOR_TESTING_IN_NETWORKING_PROCESS) is true"